roles/nginxplus/files/fail2ban/nginx-bad-httpbots.conf

```diff
@@ -0,0 +1,8 @@
+i[nginx-f_inclusive]
+enabled = true
+port    = http,https
+filter  = nginx-f_inclusive
+logpath = /var/log/nginx/access.log
+maxretry = 3
```

Comment:
Can we increase the maxretry or decrease the findtime. 
Have you set the maxretry and the findtime based on the sequence of the queries you identifies in the logs?

Reply:
It was deliberate aggressive at first in the hops that it would squeal.